nagios/README.md

```diff
@@ -51,39 +101,99 @@ define contact{
+  alias           Admin
+  email           admin@example.com
   address1        +155512312
 }
 
-define contactgroup{
+# Contact group for Wavefront user definition
+define contactgroup {
   contactgroup_name  wavefront
   alias              Notifications send to wavefront
   members            wfuser
 }
+```
+
+### Events
+#### Using CSP API Token
+```
+define command {
+    command_name nagios-to-wavefront-host
+    command_line /usr/local/nagios/libexec/nagios-events-wf.py --type '$NOTIFICATIONTYPE$' \
+                                                        --host '$HOSTNAME$' --time '$TIMET$' \
+                                                        --msg '$HOSTOUTPUT$' \
+                                                        --wf_server [WAVEFRONT_URL] \
+                                                        --csp_api_token [CSP_API_TOKEN] \
+                                                        --csp_base_url [CSP_BASE_URL]
+}
+
 
-define command{
-  command_name nagios-to-wavefront-service
-  command_line /opt/nagios/etc/wf/nagios-events-wf.py -S \
-                                                      --type '$NOTIFICATIONTYPE$' \
-                                                      --host '$HOSTNAME$' \
-                                                      --service '$SERVICEDISPLAYNAME$' \
-                                                      --time '$TIMET$' \
-                                                      --msg '$SERVICEOUTPUT$\n$NOTIFICATIONAUTHOR$\n$NOTIFICATIONCOMMENT$' \
-                                                      [WAVEFRONT_URL] \
-                                                      [WAVEFRONT_TOKEN]
+define command {
+    command_name nagios-to-wavefront-service
+    command_line /usr/local/nagios/libexec/nagios-events-wf.py -S --type '$NOTIFICATIONTYPE$' \
+                                                        --service '$SERVICEDISPLAYNAME$' \
+                                                        --host '$HOSTNAME$' --time '$TIMET$' \
+                                                        --msg '$SERVICEOUTPUT$\n$NOTIFICATIONAUTHOR$\n$NOTIFICATIONCOMMENT$' \
+                                                        --wf_server [WAVEFRONT_URL] \
+                                                        --csp_api_token [CSP_API_TOKEN] \
+                                                        --csp_base_url [CSP_BASE_URL]
 }
+```
 
-define command{
-  command_name nagios-to-wavefront-host
-  command_line /opt/nagios/etc/wf/nagios-events-wf.py --type '$NOTIFICATIONTYPE$' \
-                                                      --host '$HOSTNAME$' \
-                                                      --time '$TIMET$' \
-                                                      --msg '$HOSTOUTPUT$' \
-                                                      [WAVEFRONT_URL] \
-                                                      [WAVEFRONT_TOKEN]
+#### Using CSP Oauth
```

Comment:
#### Using the Credentials of a Server to Server OAuth App in VMware Cloud Services